pom.xml

```diff
@@ -1,197 +1,253 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please check in again with the same XML formatter settings, so that only real differences appear in PR.

I think the main difference is 4 spaces vs. the original 2 for indentation.